python/README.md

```diff
@@ -9,7 +9,7 @@ If you want to use some of the optional dependencies (OpenAI is installed by def
     python -m pip install --upgrade semantic-kernel[hugging_face]
 
 of all of them:
```

Comment:
I happen to see a typo here, ('of' is Dutch for 'or'), can you change that as well?